src/App.test.tsx

```diff
@@ -1,9 +1,6 @@
-import chunk from './utilities/chunk'
+import { render, screen } from "@testing-library/react";
+import App from "./App";
 
-test('should chunk an array properly', () => {
-  const testData = Array.from(new Array(55)).map((value, index) => `element ${index}`)
-  const result = chunk(testData, 10)
-  expect(result.length).toEqual(6)
-  expect(result[0].length).toEqual(10)
-  expect(result[5].length).toEqual(5)
+test("a user interaction", () => {
+  const wrapper = render(<App />);
```

Comment:
maybe we can add a comment like:
```
// delete this test
```

or maybe more info like:

```
// delete test, we don't need to test that the App renders, but more focused on unit tests
```

Reply:
or something like that? or maybe it's not necessary and is part of the test to see how much they know about testing?

Reply:
Add a comment